packages/shared/src/utils/cfd/cfd.js

```diff
@@ -157,40 +157,76 @@ export const isLandingCompanyEnabled = ({ landing_companies, platform, type }) =
     const onfido_status = account_status?.authentication?.identity?.services?.onfido?.status;
     const manual_status = account_status?.authentication?.identity?.services?.manual?.status;
 
     const acknowledged_status = ['pending', 'verified'];
+    const failed_cases = ['rejected', 'expired', 'suspected'];
+
+    const poa_not_submitted = poa_status === 'none';
+    const poa_acknowledged = acknowledged_status.includes(poa_status);
+    const need_poa_submission = !poa_acknowledged;
+    const need_poa_resubmission = failed_cases.includes(poa_status);
+    const poa_verified = poa_status === 'verified';
+
+    const poi_not_submitted = poi_status === 'none';
+    const poi_or_poa_not_submitted = poa_not_submitted || poi_not_submitted;
+
+    //vanuatu
 
+    const poi_verified_for_vanuatu = onfido_status === 'verified' || manual_status === 'verified';
     const poi_acknowledged_for_vanuatu =
         (onfido_status && acknowledged_status.includes(onfido_status)) ||
         (manual_status && acknowledged_status.includes(manual_status));
 
+    const poi_pending_for_vanuatu =
+        ((onfido_status && onfido_status === 'pending') || (manual_status && manual_status === 'pending')) &&
+        !poi_verified_for_vanuatu;
+
     const need_poi_for_vanuatu = !poi_acknowledged_for_vanuatu;
+    const poi_not_submitted_for_vanuatu =
+        onfido_status && onfido_status === 'none' && manual_status && manual_status === 'none';
+    const poi_resubmit_for_vanuatu =
+        ((onfido_status && failed_cases.includes(onfido_status)) ||
+            (manual_status && failed_cases.includes(manual_status))) &&
+        !poi_not_submitted_for_vanuatu &&
+        !poi_verified_for_vanuatu;
 
-    const poi_acknowledged_for_bvi_labuan =
+    const poi_poa_verified_for_vanuatu = poi_verified_for_vanuatu && poa_verified;
+
+    //bvi-labuan-maltainvest
+    const poi_acknowledged_for_bvi_labuan_maltainvest =
         (idv_status && acknowledged_status.includes(idv_status)) ||
         (onfido_status && acknowledged_status.includes(onfido_status)) ||
         (manual_status && acknowledged_status.includes(manual_status));
-    const need_poi_for_bvi_labuan = !poi_acknowledged_for_bvi_labuan;
-
-    const poa_acknowledged = acknowledged_status.includes(poa_status);
-    const poi_acknowledged = acknowledged_status.includes(poi_status);
-    const poa_poi_verified = poa_status === 'verified' && poi_status === 'verified';
-
-    const need_poa_submission = !poa_acknowledged;
 
-    const idv_acknowledged = idv_status && acknowledged_status.includes(idv_status);
+    const need_poi_for_bvi_labuan_maltainvest = !poi_acknowledged_for_bvi_labuan_maltainvest;
+    const poi_not_submitted_for_bvi_labuan_maltainvest =
+        idv_status &&
+        idv_status === 'none' &&
+        onfido_status &&
+        onfido_status === 'none' &&
+        manual_status &&
+        manual_status === 'none';
```

Comment:
```suggestion
    const poi_not_submitted_for_bvi_labuan_maltainvest =
        idv_status &&
        onfido_status &&
        manual_status &&
        [idv_status, onfido_status, manual_status].includes('none');
```

Btw, is the initial falsy check required?